packages/blocks/src/page-block/default/utils.ts

```diff
@@ -68,12 +68,15 @@ export function getBlockEditingStateByPosition(
       }
       return false;
     };
-
+    const copyRect = (rect: DOMRect) => {
+      const { top, right, bottom, left, width, height, x, y } = rect;
+      return { top, right, bottom, left, width, height, x, y };
+    };
     if (hasOptionBar(block)) {
       const currentHoverDom =
         block.type === 'image' ? hoverDom?.querySelector('img') : hoverDom;
       blockRect = currentHoverDom?.getBoundingClientRect() as DOMRect;
-      detectRect = { ...blockRect } satisfies DOMRect;
+      detectRect = copyRect(blockRect) as DOMRect;
```

Comment:
why does it work?

Reply:
https://stackoverflow.com/questions/42713229/getboundingclientrect-object-properties-cannot-be-copied

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Enumerability_and_ownership_of_properties#querying_object_properties